bugsnag-android-core/src/main/java/com/bugsnag/android/ForegroundDetector.java

```diff
@@ -29,14 +29,19 @@
      *
      * @return whether the application is in the foreground or not
      */
-    boolean isInForeground() {
-        ActivityManager.RunningAppProcessInfo info = getProcessInfo();
+    @Nullable
+    Boolean isInForeground() {
```

Comment:
Javadoc for the method needs updating -- although this looks like it might have been out of date beforehand too!

Reply:
Yes, it looks like it was out of date beforehand - I've updated the Javadoc so this should now be ready for re-review.